pom.xml

```diff
@@ -4,18 +4,43 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.dsingley</groupId>
-    <artifactId>jwt</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <groupId>com.dsingley.jwt</groupId>
```

Comment:
Doesn't this mean you have to do another Jira ticket? Suggest using just com.dsingley (like we use org.kiwiproject). This lets you publish any artifact under that group.

Reply:
no Jira ticket, just a DNS entry, and I think I like it this way

Reply:
Ok, if you like to make things more difficult, go for it!